packages/components/menu/src/sub-menu.ts

```diff
@@ -212,7 +212,11 @@ export default defineComponent({
     const doDestroy = () =>
       vPopper.value?.popperRef?.popperInstanceRef?.destroy()
 
-    const handleCollapseToggle = (value: boolean) => !value && doDestroy()
+    const handleCollapseToggle = (value: boolean) => {
```

Comment:
Omitting the curly braces indicates a default return value, but since the function here doesn't have a return value, I think it's more appropriate to keep it as is.